src/main/java/com/revature/services/CampusService.java

```diff
@@ -10,30 +10,67 @@
 import com.revature.repository.BuildingRepository;
 import com.revature.repository.CampusRepository;
 
+/**
+ * The Class CampusService.
+ * Service that talks to the repository
+ * to execute any actions involving
+ * the database.
+ * 
+ * It either gets building objects 
+ * or campus objects from the database.
+ * 
+ *  @author 1811-Java-Nick | 12/27/2018
+ */
 @Service
 public class CampusService {
 
+	/** The campus repo. */
 	CampusRepository campusRepo;
+	
+	/** The building repo. */
 	BuildingRepository buildingRepo;
 	
+	/**
+	 * Instantiates a new campus service.
+	 *
+	 * @param campusRepo the campus repo
+	 * @param buildingRepo the building repo
+	 */
 	@Autowired
 	public CampusService(CampusRepository campusRepo, BuildingRepository buildingRepo) {
 		super();
 		this.campusRepo = campusRepo;
 		this.buildingRepo = buildingRepo;
 	}
 
+	/**
+	 * Gets the list of campuses.
+	 *
+	 * @return the campuses
+	 */
 	public List<Campus> getCampuses() {
 		return campusRepo.findAll();
 	}
 	
+	/**
+	 * Gets the building by the building id.
+	 *
+	 * @param id the building id
+	 * @return the building
```

Comment:
Needs clearer documentation of what is returned.

Example: building with provided ID or null if there is no building with the provided ID.